ansible/playbook/README.md

```diff
@@ -24,16 +24,22 @@ $ ansible-playbook playbook/hetzner-create-hcloud-server.yml
 
 Variables used:
 
-| Variable | Required | Default | Meaning |
-| --- | --- | --- | --- |
-| vm_name | x | | Name of the VM to be created at hetzner. |
-| server_type | | cx31 | Server type. The list can be obtained using `hcloud server-type list`. Usually cx31 |
-| vm_image | | centos-7 | Hetzner image to be used as source (centos-7,...) | 
-| salt_text | x | | Salt and pepper. |
-| password_text | x | | Password of the created user |
-| hetzner_context_name |  | Context name for hcloud |
-| vm_delete |  | | Deletes the VM prior to creating it |
+| Variable | Required | Default | Prompt | Meaning |
+| --- | :---: | :---: | :---: | --- |
+| vm_name | x | | x | Name of the VM to be created at hetzner. |
+| server_type | | cx31 |  | Server type. The list can be obtained using `hcloud server-type list`. Usually cx31 |
+| vm_image | | centos-7 |  | Hetzner image to be used as source (centos-7,...) | 
+| salt_text | x | | x | Salt and pepper. |
+| password_text | x | | x | Password of the created user. |
+| hetzner_context_name | x | | x | Context name for hcloud. |
+| vm_delete |  | False |  | Deletes the VM prior to creating it. |
 
 ```bash
 $ ansible-playbook playbook/hetzner-create-hcloud-server.yml
-```
\ No newline at end of file
+```
+
+Optionally, using environment variables:
+
+```bash
+$ ansible-playbook playbook/hetzner-create-hcloud-server.yml -e '{ "vm_name": "my-name" }' -e '{ "salt_text": "<my salt>" }' -e '{ "password_text": "<my password>" }' -e '{ "hetzner_context_name": "<context name>" }' -e '{ "vm_delete": True }' -e '{ "pass_store_dir": "/home/<my home>/.my-password-store" }'
```

Comment:
Is there an alternative to `{}` as passing a parameter using JSON syntax is error prone and could be better handled using `-e key=value` ?

Reply:
Yes, we can use the form `-e key=value` but this has some issues that's why I purposed using json form. Booleans, for instance, are treated as String if we use `-e vm_delete=True` and it won't have the desired effect in the playbook. With booleans it's required to use the json form `-e '{ "vm_delete": True }'`.